Orange/widgets/visualize/utils/heatmap.py

```diff
@@ -405,7 +405,7 @@ def setHeatmaps(self, parts: 'Parts') -> None:
         for i, rowitem in enumerate(parts.rows):
             if rowitem.title:
                 item = QGraphicsSimpleTextItem(rowitem.title, parent=self)
-                item.rotate(-90)
+                item.setTransform(item.transform().rotate(-90))
```

Comment:
You could consider also "item.setRotation(-90)" if that's the only transformation needed.